### ISSUE TITLE
NVDA/Chrome doesn't speak 'Use arrow keys...' aria lable

### ISSUE THREAD

**Issue by corinagum** [microsoft/BotFramework-WebChat#3758]
## Version
4.12 pre-release

## Describe the bug

When using NVDA and Chrome, the aria-label 'Use arrow keys' is not spoken when focused on 'End of transcript'

## Steps to reproduce

1. Load Web Chat in Chrome
2. Type any activity
1. Turn on NVDA
1. Press Tab two times from sendbox
1. NVDA says 'End of transcript' but not 'Use arrow keys...'

## Expected behavior

Similar to Narrator x Edge, 'Use arrow keys...' aria-label should be spoken

[Bug]

**Comment by SSanjeevi** [comment]
We also need this issue to be fixed, please prioritize
